pt/utils.c

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2022 Intel Corporation.
+
+#include "utils.h"
+
+#define BIT(nr)                 (1UL << (nr))
+#define RTIT_CTL_DISRETC        BIT(11)
+#define RTIT_CTL_TSC_EN         BIT(10)
+#define PT_CTL_DISRETC RTIT_CTL_DISRETC
+#ifndef PT_CTL_TSC_EN
+#define PT_CTL_TSC_EN RTIT_CTL_TSC_EN
+#define PT_CTL_DISRETC RTIT_CTL_DISRETC
+#endif
+
+#define PT_PMU_DIR "/sys/devices/intel_pt/"
+
+int seekPck_w_lib(enum pt_packet_type pt_type, uint64_t *bufEv, long bufsize)
+{
+	struct pt_packet_decoder *decoder;
+	struct pt_config config;
+	uint64_t offset;
+	int r_val = -1;
+	int errcode;
+	struct pt_packet packet;
+	memset(&config, 0, sizeof(config));
+	pt_config_init(&config);
+	config.begin = bufEv;
+	config.end = bufEv + bufsize;
+
+	decoder = pt_pkt_alloc_decoder(&config);
+	if (!decoder) {
+		printf("pt_pkt_alloc_decoder is failed!\n");
+		r_val = -1;
+		return r_val;
+	}	else {
```

Comment:
Replace the tab with a space in the front of 'else', please.
BTW, the else is not necessary if there's a 'return' in the statement of if.